doc/guide/author/topics.xml

```diff
@@ -4601,7 +4614,360 @@ displayed line, and there are no <c>\\</c>s.  Use <c>\amp</c> to mark the alignm
             </li>
 
         </dl></p>
+    </subsection>
+
+    <subsection xml:id="subsec-img-desc-advice">
+        <title>Advice for Writing Image Descriptions</title>
+        <author>Oscar Levin</author>
+        <p>
+          Acknowledgement: the following content is based on a presentation delivered by Michael Cantino in December 2022.  Michael has served as the BVIS specialist for Oregon's K-12 public schools, an accessibility specialist at Portland Community College, and is a Library of Congress certified Braille Transcriber. Many thanks to Michael for sharing his expertise with the <pretext/> community.
+        </p>
+
+        <p>
+          Accessibility standards dictate that all (non-decorative) images included in websites are accompanied by a text description which a screen reader can read aloud to a non-sighted user.  This <term>alternative text</term> (or <term>alt text</term>) is also displayed in place of the image if the image fails to load.  Essentially alt text is a brief description of what the image is meant to convey. Of course, determining how to concisely convey information of an image can be a challenge for any image, and this is especially true for images you would find in a mathematics or science textbook.  The following suggestions and examples are intended to help an author create quality descriptions of their images.
+        </p>
+
+        <p>
+          First, a few technical considerations.  The usual way a non-sighted user will access a pretext book is through a <pretext />-generated <init>HTML</init> webpage.  Screen readers will read the text contained in the <attr>alt</attr> attribute of the <tag>img</tag> element on the page. (<pretext /> uses the text the author provides in the <tag>shortdescription</tag> element to create this content.)  Technical limitations of the <attr>alt</attr> attribute and most screen reader software mean that descriptions presented in this way must conform to particular specifications.
+        </p>
+
+        <p>
+          The content of an alt-text description must be plain text and should not exceed a total of 125 characters (although the exact character limit is a matter of some debate). Some screen readers will read more than this, but others will simply stop reading at this point. Further, it is more complicated to get screen readers to navigate alt text compared to how they would navigate the rest of text on the page.  Since the alt text is plain text, it cannot include text styles, links, MathML, or other markup.
```

Comment:
This is where I discuss the limitations.  It would be easy to add: "For this reason, PreTeXt does not allow any markup (no `<m>` or `<em>` or anything of the sort inside a `<shortdescription>`."  However, I tried to keep most of this generic, thinking that it might one day be useful for other purposes; WeBWorK folks were asking about how to write alt-text as well, for example